odbc_fdw.c

```diff
@@ -421,6 +476,15 @@ odbcGetOptions(Oid foreigntableid, char **svr_dsn, char **svr_database, char **s
     if (!*svr_dsn)
         *svr_dsn = NULL;
 
+    if (!*svr_driver)
+        *svr_driver = NULL;
+
+    if (!*svr_host)
+        *svr_host = NULL;
+
+    if (!*svr_port)
+        *svr_port = NULL;
```

Comment:
okay, just saw an outdated comment. Anyway, I'm 100% for removing code that does nothing but pretends to do something unclear.